src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,94 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.page {
+  width: 198px;
+  height: 406px;
+  border: 1px solid #F3F3F3;
+  border-radius: 5px;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+
+  &-image {
+    background-image: url(../images/imac.jpeg);
+    background-repeat: no-repeat;
+    width: 160px;
+    height: 134px;
+    margin-left: 3px;
```

Comment:
not a good approach to position img
please find another way for doing this

Reply:
hello, I have no idea, I was tried to fix it 3 days, that's why I did it(( help me, please, with it))
![image](https://github.com/mate-academy/layout_product-cards/assets/156832672/cadefde5-e263-49ba-9527-115ad88bf272)